x/auth/ante/fee.go

```diff
@@ -117,6 +117,12 @@ func (dfd DeductFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bo
 		}
 	}
 
+	var events sdk.Events
+	events = append(events, sdk.NewEvent(sdk.EventTypeTx,
+		sdk.NewAttribute(sdk.AttributeKeyFee, feeTx.GetFee().String()),
```

Comment:
Thanks for the PR Colin!

This is the total fee payed by the tx, represented as a coin/denom?
I'm trying to test this with Hermes and will provide feedback on how easy is it for the relayer to use the fee info.

As a side-note, I'm trying to rebuild my local gaiad using this branch, and was wondering if there any instructions for doing that. I'm patching my gaia `go.mod` but not sure that's the right approach.

Reply:
Yes this is the total fee, but the fee may be multiple coins. Each coin would be represented by denom/amount. An empty coin set returns an empty string

Using the SDK's `simd` binary is another alternative to `gaiad`. You can run `make build` or `make install` depending on your usage

Reply:
Yes, this is the entire fee. In the case of the Hub, it would be `Nuatom`, but could be multiple coins as @colin-axner pointed out, e.g. `Nuatom,Yphoton`